includes/class-amp-travel-blocks.php

```diff
@@ -277,7 +277,7 @@ public function render_block_activity_list( $attributes ) {
 					</div>';
 		}
 		$output .= "<div class='travel-overflow-container'>
-							<div class='flex justify-center p1 md-px1 mxn1'>";
+							<div class='flex flex-wrap flex-auto justify-center p1 md-px1 mxn1'>";
```

Comment:
These classes allow the circles of activities to wrap to the next line if they fill the line.